### PR TITLE
Add vagrant alternative to run test playbooks

### PR DESCRIPTION
**Pull request by seocam**

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Ansible FreeIPA Test Playbooks
+
+This directory contains a series of playbooks used to test the idempotency of roles provided by ansible-freeipa.
```
Comment (rjeffman) [inline tests/README.md]: I think changing "the idempotency of roles" to "modules and roles" make this more correct, as we test the modules functionality and not only idempotency.

File: tests/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+
+## Requirements
+
+In order to run those tests you will need to have `vagrant` installed with the plugins `vagrant-hostmanager` and `vagrant-libvirt`.
```
Comment (rjeffman) [inline tests/README.md]: Don't know how to rephrase it, but `vagrant` is not needed for the tests, it is needed for the automation being created.

File: tests/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+On Fedora those can be installed with the following command: `dnf install vagrant vagrant-hostmanager vagrant-libvirt`
+
+
+## Topology
+
+The topology implemented by the vagrant file consists in:
```
Comment (rjeffman) [inline tests/README.md]: I think "consists in" should be "consists of".

**Comment by rjeffman** [comment]
There should be something about setting VAGRANT_DEFAULT_PROVIDER, as you can't pass `--provider` for the `run-test-in-vagrant.sh` script.

**Comment by seocam** [comment]
Closing for now. I'm thinking of automating in using virinstall/virsh as we are currently doing in development.

**Pull request closed without merging**